sympy/core/exprtools.py

```diff
@@ -1098,6 +1098,25 @@ def handle(a):
     return terms.func(*[handle(i) for i in terms.args])
 
 
+def _factor_sum_int(expr, **kwargs):
+    from sympy.core.exprtools import factor_terms
```

Comment:
Is this the function right below?

Reply:
ahh...leftover from when the function wasn't in the same file. I will remove that import.